### DISCUSSION TITLE
Can I import a .js?

### DISCUSSION THREAD

**Discussion by timeanddoctor** [Kitware/trame#199] [Q&A] [answered]
I tend to use a js file such as : <script src="https://cdn.bootcdn.net/ajax/libs/mobile-detect/1.4.5/mobile-detect.min.js"></script>

I find the js_call method but I dont know how to import it first. Thank you!

**Comment by jourdain** [comment]
```python
server = get_server()
server.state.trame__scripts += ["https://cdn.bootcdn.net/ajax/libs/mobile-detect/1.4.5/mobile-detect.min.js"]
```

**Comment by jourdain** [comment, marked as answer]
`js_call` won't help you here since it is aimed to call function on a vue component and not eval a JS expression.

**Reply by timeanddoctor** [reply]
Thanks Professor Jourdain.
How can I use the imported js.  
Can I insert the <script> with python?
Thanks.

**Reply by jourdain** [reply]
> Can I insert the <script> with python?

You just did with `server.state.trame__scripts += [...]`

What do you want to do with that script? You can create your own vue.js component and expose any method you want and call them using `js_call`. There is no limit on what you can do.
